plugins/minimator/tests/profile_default.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
+from conans import ConanFile, tools
+
+
+class CloeTest(ConanFile):
+    name = "cloe-minimator-test"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "virtualenv", "virtualrunenv"
+
```

Comment:
I see your point. I would prefer if there was a way to say use `cloe-engine` as override for any conflicts, or directly access the dependencies of `cloe-engine`:

```
requires(self):
  self.requires(f"cloe-engine/{self.version}@cloe/develop")
  self.requires(self.requires["cloe-engine"].requires["boost"], override=True)
```

Probably this is not possible, but it's semantics I wish were possible. We should create an issue at conan and see if there is a way to do something like this.

Reply:
I agree, For time being, I wonder if we would need in addition to 
```
    default_options = {
        "cloe-engine:server": False,
    }
```
in the requirements
```
if self.options["cloe-engine"].server:
    self.requires("boost/[~=1.69]", override=True)
```
Otherwise, the users will run into version conflicts. The same applies to `plugints/vtd/tests/profile_default.py`,

Reply:
Done.